### PR TITLE
7.5 Testing Update

### PR DESCRIPTION
**Pull request by MichaelThompson**
Updated tests to work with 7.5 and the v4 schema instead of v3.

### CODE REVIEW

**Comment by MichaelThompson** [comment]
@JeffHarkavy

File: cmdlet-tests-a2a.ps1 [hunk 1]
```diff
@@ -310,13 +310,27 @@ try {
    Write-Host "Create-AccessRequest"
    #region AccessRequest
    $selectedAccount = FindAssetAccount(0)
-   $selectedAsset = Get-SafeguardAsset -AssetToGet $selectedAccount.AssetId
+   $selectedAsset = Get-SafeguardAsset -AssetToGet $selectedAccount.Asset.Id
+   
+   Write-Host "Create-AccessRequest 2"
```
Comment (JeffHarkavy) [inline cmdlet-tests-a2a.ps1]: Do you still need the debug output for WIP? (here & below)

**Review by JeffHarkavy — APPROVED** [review]

File: cmdlet-tests-a2a.ps1 (hunk 1 above)
Comment (MichaelThompson) [reply cmdlet-tests-a2a.ps1]: No, I don't. Thanks for noticing

**Comment by MichaelThompson** [comment]
@JeffHarkavy  It appears that only you have permission to merge this PR.

**Pull request merged**